src/OrchardCore.Modules/OrchardCore.Users/Views/UserRolesMeta.cshtml

```diff
@@ -0,0 +1,21 @@
+@using OrchardCore.Security.Services
+@model SummaryAdminUserViewModel
+
+@inject IRoleService RoleService
+
+@{
+    var roleNames = await RoleService.GetRoleNamesAsync();
+}
+
+<div data-bs-toggle="tooltip" title="@T["The roles this user has"]">
+    @foreach (var normalizedRoleName in Model.User.RoleNames)
+    {
+        var roleName = roleNames.FirstOrDefault(roleName => roleName.Equals(normalizedRoleName, StringComparison.OrdinalIgnoreCase));
```

Comment:
@jtkech  Either we use `OrdinalIgnoreCase` or we normalize the names before we do a look up here since we store the normalized name in the user's object

Reply:
@MikeAlhayek Okay, so above line 7 just use `GetNormalizedRoleNamesAsync()` in place of `GetRoleNamesAsync()`.

And then remove `OrdinalIgnoreCase`.

Reply:
done. I was not aware of that method :)